tests/framework/AnalyticModels/projectile.py

```diff
@@ -87,14 +87,15 @@ def run(raven, inputs):
           'y0': get_from_raven(raven,'y0', 0),
           'v0': get_from_raven(raven,'v0', 1),
           'ang': get_from_raven(raven,'v0', 45),
-          'timeOption': get_from_raven(raven,'v0', 0)}
+          'timeOption': get_from_raven(raven,'timeOption', 0)}
   res = main(vars)
   raven.x = res['x']
   raven.y = res['y']
   raven.t = res['t']
   raven.r = res['r'] * np.ones(len(raven.x))
   raven.v = res['v']
   raven.a = res['a']
+  raven.timeOption = vars['timeOption']
```

Comment:
use **res['timeOption']**?

Reply:
I used vars because theoretically it should come from the input file (raven)